src/translations/fr/LC_MESSAGES/messages.po

```diff
@@ -1232,7 +1235,21 @@ msgstr ""
 "ayant été inclues dans le plus grand nombre de decks gagnants depuis. </p> <p> Tous les %(hunting_grounds)s et, "
 "parfois, quelques vampires de basse capacité, ont été omis par commodité. </p>"
 
-#: src/templates/best-cards/index.html:43
+#: src/templates/best-cards/index.html:21
+#, python-format
+msgid ""
+"<ul> <li>%(master)s</li> <li>%(political)s</li> <li>%(no_discipline)s</li> <li><i>i</i> %(animalism)s</li> "
+"<li><i>a</i> %(auspex)s</li> <li><i>c</i> %(celerity)s</li> <li><i>d</i> %(dominate)s</li> <li><i>f</i> "
+"%(fortitude)s</li> <li><i>n</i> %(necromancy)s</li> <li><i>o</i> %(obfuscate)s</li> <li><i>p</i> %(potence)s</li> "
+"<li><i>r</i> %(presence)s</li> </ul>"
+msgstr ""
```

Comment:
So, this is the kind of section I was hoping to avoid so as to make things easier on non-developer translators.

Reply:
I'll rework it using `{{ aus }}` styling